src/AspNetCoreModuleV2/RequestHandlerLib/PipeOutputManager.cpp

```diff
@@ -126,12 +122,10 @@ HRESULT PipeOutputManager::Start()
 
     if (m_hErrThread == NULL)
```

Comment:
RETURN_LAST_ERROR_IF_NULL

Reply:
Actually this check is wrong. It should be checking for invalid handle.

Reply:
nvrm I'm wrong.

Reply:
https://docs.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-createthread#return-value